README.md

```diff
@@ -108,32 +123,32 @@ To deploy this app follow the steps below:
 
 	https://docs.djangoproject.com/en/1.3/howto/deployment/modwsgi/
```

Comment:
Our app is in Django 1.6
Link to documentation should also be to Django 1.6 docs